documentation/platyps-help.md

```diff
@@ -0,0 +1,227 @@
+
+#### NOTE
+
+Some cmdlets have examples containing rows and columns of information after the PowerShell commands and remarks. For example:
+
+    ### --------------------------  Example 1: Get capabilities for the current subscription for a region  --------------------------
+    @{paragraph=PS C:\\\>}
+    
+    ```
+    PS C:\> Get-AzureRmSqlCapability -LocationName "Central US"
+    ```
+
+    This command returns the capabilities for SQL Database on the current subscription for the Central US region.
+
+    Location                : Central US
+    Status                  : Available
+    SupportedServerVersions : {12.0, 2.0}
+
+However, the format of these rows and columns are not preserved when platyPS generates the MAML. In order to avoid this issue, please move the rows and columns into the code block of the example; this is how help for core PowerShell modules is written.
+
+After making these changes, the resulting markdown would look like the following:
+
+    ### --------------------------  Example 1: Get capabilities for the current subscription for a region  --------------------------
+    @{paragraph=PS C:\\\>}
+    
+    ```
+    PS C:\> Get-AzureRmSqlCapability -LocationName "Central US"
+    
+    Location                : Central US
+    Status                  : Available
+    SupportedServerVersions : {12.0, 2.0}
+    ```
+
+    This command returns the capabilities for SQL Database on the current subscription for the Central US region.
+
+Making these changes will allow the table to preserve the format when the MAML is displayed in the PowerShell window.
+
+### Updating markdown
+
+Once you have created the markdown files for each cmdlet, you must make sure to keep them up to date based on changes to the cmdlets.
+
+If any changes are made to a cmdlet (*i.e.,* add/remove parameters, add/remove parameter sets, etc.), the corresponding markdown can be updated through reflection by using the following command
+```powershell
+Update-MarkdownHelp -Path $path
+```
+where `$path` is the path to the folder containing all of the markdown help.
+
+If a cmdlet is added to your module, then the above command will not work; you must first add the corresponding markdown file by running the following command (also used in the previous section):
+```powershell
+New-MarkdownHelp -OutputFolder $output -Module $module -WithModulePage
+```
+
+However, an easier way to update the markdown files to handle **both** changes to cmdlets and new cmdlets is to use the following command:
+```powershell
+Update-MarkdownHelpModule -Path $path
+```
+where `$path` is the path to the folder containing all of the markdown help.
+
+This is the **recommended** command because it updates all the files in the specified folder based on the cmdlets as loaded into your current session.
+
+### Updating MAML
```

Comment:
In general, everywhere where we have MAML, I would have 'Xml Help (MAML)'  again, MAML is not well known as the name of the powershell help schema